app/assets/stylesheets/application.scss

```diff
@@ -1,15 +1,13 @@
 @import "bootstrap";
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *= require_tree .
- *= require_self
- */
+
+.container {
+  content: center;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.center-title {
+  text-align: center;
+}
```

Comment:
If these styles are intended to be used across any page of the app, defining them in this file is OK, just keep in mind that if you find yourself adding styles that will be only applicable on specific pages then it's better to define the CSS rules in their own stylesheet